### PR TITLE
fixed the build on x86_64 with a updated release

### PR DESCRIPTION
**Pull request by Paradoxianer**
I had to create a new release thats why the checjsun changed

### CODE REVIEW

**Comment by Begasus** [comment]
Thanks, let's check this on the Beta first :)

File: haiku-apps/scriptureguide/scriptureguide-0.9.1.recipe [hunk 1]
```diff
@@ -8,9 +8,9 @@ COPYRIGHT="2004 Jan Bungeroth
 	2005-2019 ScriptureGuide Team"
 LICENSE="GNU GPL v2
 	MIT"
-REVISION="2"
+REVISION="3"
 SOURCE_URI="https://github.com/Paradoxianer/ScriptureGuide/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="d45a9be201e829a6c6c8e0cece33297899d89061f77c08e8f0a0c1a68a23c3c4"
+CHECKSUM_SHA256="386f8ab254ec12e8e9e1b79dcb58f6b4084ecf5901370ef951c21ace6b127ba7"
```
Comment (Begasus) [inline haiku-apps/scriptureguide/scriptureguide-0.9.1.recipe]: Maybe add:
```diff
+SOURCE_FILENAME="ScriptureGuide-v$portVersion.tar.gz"
```

**Comment by Begasus** [comment]
Build is OK on Beta :+1:

**Comment by Paradoxianer** [comment]
Stupid me forgot the make bindcatalogs #4670

**Comment by Begasus** [comment]
Time for another update then :package: ;)

**Comment by Begasus** [comment]
Maybe wait a bit :) For now it's working, think you need to add a few words to translate :)
Scripture-Guide

**Pull request merged**